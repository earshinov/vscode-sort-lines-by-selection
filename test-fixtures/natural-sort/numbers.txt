[2.11]
[10]
[2]
[-2.11]
[-0]
[-2.5]
[0]
[-0.5]
[-2]
[-10]
[0.5]
[2.5]
