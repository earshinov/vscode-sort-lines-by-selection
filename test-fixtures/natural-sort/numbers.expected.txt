[-10]
[-2.5]
[-2.11]
[-2]
[-0.5]
[-0]
[0]
[0.5]
[2]
[2.11]
[2.5]
[10]
